program/verilator-verification.ppl.md

```diff
@@ -0,0 +1,33 @@
+# OpenHW Preliminary Project Proposal: Verilator verification for CORE-V MCU SoC
```

Comment:
Suggest "Verilator modeling for CORE-V MCU and FPGA SoC"

The software side of this is not verification. And it would be useful to have both MCU and FPGA variants supported.